lab-ron/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const program = require('./lib/read-files.js');
+
+module.exports = exports = {};
```

Comment:
@Rubiksron Any reason this module.exports assignment was added?  Since nothing from this file needs to be exported, no need for this line at this point.